tox.ini

```diff
@@ -5,7 +5,7 @@ skip_missing_interpreters = True
 [testenv]
 commands =
     python --version
-    pytest tests --cov-report term-missing --cov-fail-under 100 --cov cornice {posargs}
+    pytest {posargs: --cov-report term-missing --cov-fail-under 100 --cov cornice}
```

Comment:
Everything behind `posargs:` is the default value which can be overridden - this is necessary to be able to select a single test.

Reply:
oH I had no idea of this, thanks!